bitcoin-information/news.html

```diff
@@ -122,6 +122,7 @@ <h2 id="news"><a href="#news">News Sites:</a></h2>
             <li><a href="http://coinjournal.net/" title="CoinJournal" target="_blank" rel="noopener">CoinJournal</a></li>
             <li><a href="https://www.merklereport.com/" title="Merkle Report" target="_blank" rel="noopener">Merkle Report</a></li>
             <li><a href="https://messari.io/" title="Messari" target="_blank" rel="noopener">Messari</a></li>
+            <li><a href="https://finbold.com/" title="Finbold" target="_blank" rel="noopener">Finbold</a></li>
```

Comment:
please move this up 2 lines to line 123 so that it's in alphabetical order

Reply:
Done.